cleanlab/rank.py

```diff
@@ -557,7 +557,7 @@ def get_confidence_weighted_entropy_for_each_label(
 
 
 def get_knn_distance_ood_scores(
-    features: np.array, nbrs: NearestNeighbors, k: int = None
+    features: np.array, nbrs: NearestNeighbors = None, k: int = None
```

Comment:
nit: favor long form > short form, i.e. use 'neighbors'. That being said, calling this argument `knn` might be even clearer.